### PR TITLE
 Admin Viewing items: Closes 45

### LINKED ISSUE #45: 24: Admin Viewing items
As an Admin 
When I visit "/admin/dashboard" 
Then I should see a link for viewing all items 
And when I click that link 
Then my current path should be "/admin/items" 
Then I should see a table with all items (active and inactive) 
And each item should have:

- A thumbnail of the image
- Title that links to the item
- Description
- Status
- Actions (Edit)

### PR DESCRIPTION
**Pull request by swdonovan**
added items index and show page for an admin and then linked the containers of items indexes to be a link to the item show page and the program decyphers the difference between if you are an admin or a guest. broke out the html code for all items to a partial so it could be reused again for the admin portion of the indexes. and then routed with passing test

### CODE REVIEW

**Review by jimszalew — APPROVED** [review]
solid work here.

File: app/views/items/index.html.erb [hunk 1]
```diff
@@ -1,16 +1,2 @@
...
-          <h3><%= item.title %></h3>
-          <p><%= item.description %></p>
-          <p><%= item.price %></p>
-          <%= button_to "Add to Cart", carts_path(item_id: item.id), class: "btn btn-primary", id: "item-#{item.id}" %>
-      </div>
-    </div>
-    </div>
-  <% end %>
-</div>
+<%= render partial: 'all_items' %>
```
Comment (jimszalew) [inline app/views/items/index.html.erb:2]: Nice use of the partial here.

File: app/views/admin/items/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<h1 class="ProductIndexHeader">Admin - All Products</h1>
+<%= render partial: 'items/all_items' %>
```
Comment (MikelSage) [inline app/views/admin/items/index.html.erb:2]: Nice use of partials here.

**Review by MikelSage — APPROVED** [review]
Looking good. Merging in

**Pull request merged**